util/git/client.go

```diff
@@ -651,6 +651,8 @@ func (m *nativeGitClient) runCmdOutput(cmd *exec.Cmd) (string, error) {
 	cmd.Env = append(cmd.Env, "HOME=/dev/null")
 	// Skip LFS for most Git operations except when explicitly requested
 	cmd.Env = append(cmd.Env, "GIT_LFS_SKIP_SMUDGE=1")
+	// Disable Git terminal prompts in case we're running with a tty
```

Comment:
```suggestion
	// Disable Git terminal prompts in case we're running with a tty (this happens during local development)
```

Reply:
Technically, this can happen in a lot of scenarios - even when running inside Kubernetes and someone to decides to set `tty: true` for the container running the repository image binary.

Reply:
Ah, sure. Resolving